composer.json

```diff
@@ -9,12 +9,12 @@
     ],
     "require": {
         "php": ">=5.4.0",
-        "illuminate/support": "4.2.*|5.0.*|5.1.*"
+        "illuminate/support": "4.2.*|5.0.*|5.1.*|5.2.*"
```

Comment:
I don't think so, see the Composer documentation for [caret](https://getcomposer.org/doc/articles/versions.md#caret).